SimPPS/PPSPixelDigiProducer/interface/RPixLinearChargeCollectionDrifter.h

```diff
@@ -0,0 +1,46 @@
+#ifndef SimPPS_PPSPixelDigiProducer_RPix_LINEAR_CHARGE_COLLECTION_DRIFTER_H
+#define SimPPS_PPSPixelDigiProducer_RPix_LINEAR_CHARGE_COLLECTION_DRIFTER_H
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "SimPPS/PPSPixelDigiProducer/interface/RPixSignalPoint.h"
+#include "SimPPS/PPSPixelDigiProducer/interface/RPixEnergyDepositUnit.h"
+
+class RPixLinearChargeCollectionDrifter
+{
+public:
+  RPixLinearChargeCollectionDrifter(const edm::ParameterSet &params, uint32_t det_id);
+  std::vector<RPixSignalPoint> Drift(const std::vector<RPixEnergyDepositUnit>  &energy_deposition);
+
+private:
+  std::vector<RPixSignalPoint> temp_;
+
+  std::vector<double> charge_cloud_sigmas_vect_; 
+  double GeV_per_electron_;
+  int verbosity_;
+  double det_thickness_;
+  uint32_t det_id_;
+    
+  inline double GetSigma(double z) 
```

Comment:
What is the benefit of inlining this function? Since it's private and there are no friends, it can be called only from the constructor or `Drift()`, which are already in the `.cc` file.

In addition, the convention is to start method/function names with lowercase letter.